roles/openshift_facts/tasks/main.yml

```diff
@@ -64,8 +64,7 @@
     role: common
     local_facts:
       debug_level: "{{ openshift_debug_level | default(2) }}"
-      # TODO: Deprecate deployment_type in favor of openshift_deployment_type
-      deployment_type: "{{ openshift_deployment_type | default(deployment_type) }}"
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
```

Comment:
Not in my experience - it succeeds as written even if neither variable is defined. I wonder if the behavior is different between ansible versions? I don't mind rewriting it, just looked a little cleaner to me this way.